docsrc/source/install.rst

```diff
@@ -125,6 +125,32 @@ The output looks similar to this:
         pythonwithpixiedustspark20    /Users/USERNAME/Library/Jupyter/kernels/pythonwithpixiedustspark20
         pythonwithpixiedustspark21    /Users/USERNAME/Library/Jupyter/kernels/pythonwithpixiedustspark21
 
+Stash to Cloudant
+-----------------
+
+You can export the data to a Cloudant database. A supporting library cloudant-spark jar is required for the export.
+This is a manual one-time step that requires a kernel restart. Download cloudant-spark jar file from
+::
+
+    https://github.com/cloudant-labs/spark-cloudant#Binary-download
```

Comment:
Can we specify the exact location of the jar:
for Spark 1.6: https://github.com/cloudant-labs/spark-cloudant/releases/download/v1.6.4/cloudant-spark-v1.6.4-167.jar
for Spark 2.0: https://github.com/cloudant-labs/spark-cloudant/releases/download/v2.0.0/cloudant-spark-v2.0.0-185.jar